vite.config.dev.ts

```diff
@@ -43,8 +46,10 @@ async function getConfig(): Promise<UserConfig> {
         hook: 'writeBundle'
       }),
     ],
+    envDir: "udata_front/theme/gouvfr/datagouv-components",
```

Comment:
We are using the `.env` file from the components folder otherwise the components source are missing expected variables